data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -189,7 +205,10 @@ public InputStream executeStreamingQuery(String database, String command) throws
     }
 
     @Override
-    public InputStream executeStreamingQuery(String database, String command, ClientRequestProperties properties) throws DataServiceException, DataClientException {
+    public InputStream executeStreamingQuery(
+            String database,
+            String command,
+            ClientRequestProperties properties) throws DataServiceException, DataClientException {
```

Comment:
Similar to method calls, I don't think this approach to method signatures is beneficial often enough to warrant being enforced.